iceoryx_posh/test/moduletests/test_popo_base_subscriber.cpp

```diff
@@ -31,10 +31,15 @@
 using namespace ::testing;
 using ::testing::_;
 
+// anonymous namespace to prevent linker issues or sanitizer false positives
```

Comment:
I was aware of this and always have all my tests in a local anonymous namespace. This should be in the test recommendation file as well, if if is not already mentioned there.

Avoid weird linker issues, some of them seemingly silent...